bower.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "requirejs-text",
```

Comment:
It looks like `bower info requirejs/text` seems to do the right thing, and I believe the advantage of bower using git is to allow for multiple branches but having the local name be specific. So if you can update the name to be just "text", then I am ready to merge the change.